qiita_db/analysis.py

```diff
@@ -525,19 +525,70 @@ def can_be_publicized(self):
 
         Returns
         -------
-        bool
-            Whether the analysis can be publicized or not
+        bool, list
+            Whether the analysis can be publicized or not and the list of not
```

Comment:
not public -> private ?

Reply:
Also this bool and list should be two different return arguments in the docstring:

```
Returns
-------
bool
    foo bar baz
list
    foo fo foooooooo
```